src/System.Data.Common/System.Data.Common.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
+# Visual Studio 15
```

Comment:
Sure, I just edited in vs2017. It loads OK in 2015 tho, but I'll revert the change.